webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criterion-rating-edit.js

```diff
@@ -145,6 +145,13 @@ export class SakaiRubricCriterionRatingEdit extends RubricsElement {
     this.dispatchEvent(new CustomEvent('delete-rating', {detail: this.rating}));
     this.hideToolTip();
   }
+
+ openEditWithKeyboard(e){
```

Comment:
Please format this like the rest of the file. And, you're checking for 32 twice.